src/components/header/dropdown-product/dropdown-product.scss

```diff
@@ -0,0 +1,65 @@
+*{
+    color: white;
```

Comment:
Try to add a pattern for the colors, I already see #fff and white, the HEX code is more performative if you want to use it, and allocating inside a variable will help you to code faster